nl_server/util.py

```diff
@@ -0,0 +1,19 @@
+# copyright 2024 google llc
+#
+# licensed under the apache license, version 2.0 (the "license");
+# you may not use this file except in compliance with the license.
+# you may obtain a copy of the license at
+#
+#      http://www.apache.org/licenses/license-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the license is distributed on an "as is" basis,
+# without warranties or conditions of any kind, either express or implied.
+# see the license for the specific language governing permissions and
+# limitations under the license.
+
+import os
+
+
+def is_custom_dc() -> bool:
```

Comment:
Can we move this under `shared.lib`?

Reply:
`shared` gets used in many contexts (not just NL and Web server), so I think its better to keep this here?